src/client_side_reply.cc

```diff
@@ -1656,6 +1656,8 @@ clientReplyContext::cloneReply()
 
     /* do header conversions */
     buildReplyHeader();
+
+    http->al->reply = reply;
```

Comment:
Please move this initialization higher, to give all ACLs access to sent reply, including reply_header_add ACLs.